local.go

```diff
@@ -487,6 +487,8 @@ func NewPrivIdentity(suite network.Suite, port int) (kyber.Scalar, *network.Serv
 	kp := key.NewKeyPair(suite)
 	id := network.NewServerIdentity(kp.Public, address)
 	ServiceFactory.generateKeyPairs(id)
+	id.Address = address
```

Comment:
Seems useless as this is already done in `NewServerIdentity`.

Reply:
The PR is getting very small ;)

Reply:
But it's needed for https://github.com/dedis/cothority/pull/2445/files#diff-01980534bc5287f23e37d7f5af82156839e052f43a435b3da70f2a8a731aa8b6R684